apps/backend/src/logger/logger.service.ts

```diff
@@ -1,22 +1,49 @@
 import { LoggerService } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
 import axios from 'axios';
 import { WinstonModule } from 'nest-winston';
+import { Repository } from 'typeorm';
 import winston from 'winston';
+import DailyRotateFile from 'winston-daily-rotate-file';
+import { join } from 'path';
+import * as util from 'util';
+import { LogEntity } from './entities/log.entity';
+import { logFileFormat, consoleLogFormat } from './format';
 
 export class AppLogger implements LoggerService {
-  private logger;
+  private readonly logger;
 
-  constructor() {
+  constructor(
+    @InjectRepository(LogEntity) private logsRepository: Repository<LogEntity>
+  ) {
     this.logger = WinstonModule.createLogger({
-      transports: [new winston.transports.Console({})],
+      transports: [
+        new winston.transports.Console({
+          level: 'info',
+          format: consoleLogFormat
+        }),
+        new DailyRotateFile({
```

Comment:
TODO: make this optional based on the environment

Reply:
done :)